controllers/authorino_controller_test.go

```diff
@@ -299,3 +325,13 @@ func checkAuthorinoEnvVar(authorinoInstance *api.Authorino, envs []k8score.EnvVa
 		}
 	}
 }
+
+func isAuthorinoInstanceReady(authorino *api.Authorino) bool {
```

Comment:
In Authorino we went with defining a function of the Status type for this:
https://github.com/Kuadrant/authorino/blob/0a5c235f512d4be08638665518a3c4a18a9e9532/api/v1beta1/auth_config_types.go#L668

Reply:
Good idea